src/main.cpp

```diff
@@ -1,9 +1,91 @@
 #include <Arduino.h>
+#include <SPI.h>
+#include <WiFiNINA.h>
+
+#include "WiFiNINAConnect.h"
+#include "LSM9DS1.h"
+#include "LSM9DS1_registers.h"
+#include "DS18B20.h"
+#include "HTTP.h"
+
```

Comment:
If any comments encourages changes in this file (like extra/redundant parameters, consider removing them here as well)